apps/date-component/README.md

```diff
@@ -0,0 +1,82 @@
+# Date component example
+
+This tutorial explains the steps to create a date using the [hof-component-date](https://github.com/UKHomeOfficeForms/hof-component-date). There is also a further explanation of useful date validations.
+
+## Getting started
+
+Add a step that displays a date of birth and a field called date-of-birth.
+
+```
+  steps: {
+    '/date-of-birth': {
+      fields: ['date-of-birth'],
+      next: '/confirm'
+    }
+  }
+```
+
+## Adding the Date-component field
+
```

Comment:
Ehen referring to the dat component, probably best to either be precise `hof-component-date` or verbose as in 'Date Component'